changelogs/fragments/6476-new-keycloak-user.module.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - new module for keycloak users (https://github.com/ansible-collections/community.general/pull/6476).
```

Comment:
Please remove this changelog fragment. New modules must not have changelog fragments.